front/components/assistant/conversation/input_bar/InputBar.tsx

```diff
@@ -294,6 +298,10 @@ export function AssistantInputBar({
     }
   }, [isStopping, generationContext.generatingMessages, conversationId]);
 
+  useEffect(() => {
```

Comment:
I should probably move this state to context and control from the outside instead of using useEffect but this is a quick fix 🙃